clicommand/agent_start.go

```diff
@@ -1032,6 +1032,7 @@ var AgentStartCommand = cli.Command{
 		}
 
 		if len(cfg.AllowedRepositories) > 0 {
+			l.Info("Evaluating allowed repositories has been enabled")
```

Comment:
Sounds good to me. Let me rework it and update the PR.

FYI. I recall you from SYD-PIR. You were working with Marek 😉 He was my 🇵🇱 connection.

Reply:
Oh! Hey 👋 I thought you looked familiar. I do miss my old colleagues a bit 😔 How long have you been at Canva?

Reply:
After nearly 7 years in network engineering and 1.5 year as embedded SWE in ChromeOS, the time has come to make a change. Working on Infra / CI since Apr, 2023 and I'm having good fun 👍

Reply:
Nice, glad you're having fun! I took some time off between exiting SRE and coming to BK in Oct 2022, it was worth it.